apps/web/src/modules/create-proposal/components/ReviewProposalForm/ReviewProposalForm.tsx

```diff
@@ -177,12 +168,13 @@ export const ReviewProposalForm = ({
         setError(err.message)
       }
     },
-    [signer, router, addresses, proposalThreshold, votes, clearProposal]
+    [router, addresses, proposalThreshold, votes, clearProposal]
   )
 
   if (isLoading || thresholdIsLoading) return null
 
-  const tokensNeeded = proposalThreshold && proposalThreshold.toNumber() + 1
+  const tokensNeeded =
+    proposalThreshold !== undefined ? Number(proposalThreshold) + 1 : undefined
```

Comment:
Curious, is this ``proposalThreshold !== undefined`` stuff because of BigInt (ex. 0n)?

Reply:
yes 0 is a valid proposal threshold so the `proposalThreshold &&` check didn't work when moving from `BigNunber` to `BigInt`